### PR TITLE
20190529 1350

### PR DESCRIPTION
**Pull request by devinreams**
Latest based on https://github.com/mozilla-lockwise/lockwise-ios/pull/992

Looks like it really wants to translate those bundle names (CredentialProvider, Firefox Lockwise).

But this should expose the Face ID usage description as noted at https://github.com/mozilla-lockwise/lockwise-ios/issues/1009

### CODE REVIEW

File: de/lockwise-ios.xliff [hunk 1]
```diff
@@ -94,23 +94,35 @@
...
+      </trans-unit>
       <trans-unit id="hwO-wS-oS4.text">
         <source>Take your passwords everywhere</source>
         <target>Nehmen Sie Ihre Passwörter überall mit hin</target>
         <note>Class = "UILabel"; text = "Take your passwords everywhere"; ObjectID = "hwO-wS-oS4";</note>
       </trans-unit>
     </body>
   </file>
-  <file original="lockbox-ios/Common/Resources/InfoPlist.strings" source-language="en" target-language="de" datatype="plaintext">
+  <file original="lockbox-ios/Common/Resources/Strings/en.lproj/InfoPlist.strings" source-language="en" target-language="de" datatype="plaintext">
```
Comment (Pike) [inline de/lockwise-ios.xliff:108]: The `Strings/en.lproj` path insert is surprising to me.

**Review by Pike — COMMENTED** [review]
Would the paths continue to roundtrip with the `Strings/en.lproj` in them?

**Comment by devinreams** [comment]
>Would the paths continue to roundtrip with the Strings/en.lproj in them?

@Pike Yes I believe so. I think the original inclusion of the `InfoPlist.strings` was at an incorrect path in the project so after doing some cleanup and removing of redundant files (and I just tried it again) this is what the result is.

I looked at the `Localizable.strings` and its included in Xcode and configure the same way. So I'm not sure why this would change now but don't see any way to "put it back" on this side.

image


image

**Comment by Pike** [comment]
Looping in Matjaz. @mathjazz, can you take a look at this, and see if the changed file path drops many localizations in Pontoon? Might just be a single Lockwise string at this point?

**Comment by flodolo** [comment]
> Looping in Matjaz. @mathjazz, can you take a look at this, and see if the changed file path drops many localizations in Pontoon? Might just be a single Lockwise string at this point?

The update script already dropped them: it reads the strings, and inject them in the template. If the file path changed, that translation doesn't exist in the translated XLIFF. 

If you look at German, you'll see that the translation for `Lockwise` has been removed.

P.S. Reminder that Matjaz is on PTO for the next week ;-)

**Review by Pike — CHANGES_REQUESTED** [review]
OK, dug a little deeper, and this is actually a regression in @joeyg's export branch.

https://github.com/joeyg/ios-l10n-scripts/blob/fb87542e86e6ead31e44012fd929db833dfc99c3/export-locales.sh#L98 doesn't catch the new `/Strings/` path.

I'd be OK with just the few string being lost and keeping the `/Strings/` in the filepath, but I think we need a fix to at least remove the `en.lproj` in the path.

**Comment by joeyg** [comment]
@pike ahh great catch. Sorry about that

**Comment by devinreams** [comment]
>I think we need a fix to at least remove the en.lproj in the path.

OK, since this blocks additional strings being exported (we may have 1 for a future release, fairly uncommon) I will submit a change to @joeyg's fork/branch and re-submit an extract here.

**Comment by devinreams** [comment]
Revised output at #6

**Pull request closed without merging**